classes/Http/Controller/TalkController.php

```diff
@@ -291,6 +299,17 @@ public function processCreateAction(Request $req)
 
         if ($isValid) {
             $sanitized_data = $form->getCleanData();
+            $tags = explode(',', $sanitized_data['tags']);
+
+            $tag_entity = [];
```

Comment:
How about using `array_map()` here instead, and also, maybe importing `OpenCFP\Domain\Entity`, as well as using plural for the variable name?

```php
$tag_entities = array_map(function ($tag) {
    return new Entity\Tag([
        'tag' => trim($tag),
    ]);
}, $tags);
```

Alternatively, maybe just inline `$tags` and rename `$tag_entity` to `$tags`?